src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -84,4 +119,8 @@ public double straightTurnPower(){
     double turnPower = error * .15;
     return turnPower;
   }
+
+  public double driveStraightGyro(double desiredAngle){  
```

Comment:
Give this a better name. Maybe something like getAngularDriftError or getAngularDeltaError?